opentreemap/opentreemap/settings/default_settings.py

```diff
@@ -236,6 +236,9 @@
 STATSD_HOST = os.environ.get('OTM_STATSD_HOST', 'localhost')
 STATSD_CELERY_SIGNALS = True
 
+CELERY_DEFAULT_QUEUE = STACK_TYPE
```

Comment:
`StackType` remains consistent when multiple stacks are deployed within the same environment. I think we want to use `StackColor` here.